main/src/androidTest/java/cgeo/geocaching/activity/waypoint/AbstractEditWaypointActivityTest.java

```diff
@@ -12,9 +11,9 @@ protected void setUp() throws Exception {
         super.setUp();
         createWaypoint();
         getCache().addOrChangeWaypoint(waypoint, true);
-        final int waypointId = getCache().getWaypoints().get(0).getId();
-        setActivityIntent(new EditWaypointActivity_.IntentBuilder_(getInstrumentation().getContext()).geocode(getCache().getGeocode()).waypointId(waypointId).get());
-        getActivity();
+//        final int waypointId = getCache().getWaypoints().get(0).getId();
+//        setActivityIntent(new EditWaypointActivity.IntentBuilder_(getInstrumentation().getContext()).geocode(getCache().getGeocode()).waypointId(waypointId).get());
+//        getActivity();
```

Comment:
Remove or reactivate it.

Reply:
This class will be deleted eventually after #13798 is resolved and I can fix the GUI tests for good. I keep it only as reference for later test refactoring. I will comment the class accordingly, hope it is then fine to keep it for now.